azure-pipelines.yml

```diff
@@ -0,0 +1,87 @@
+trigger:
+  branches:
+    include:
+    - '*'
```

Comment:
don't CI build every branch, that would actually mean two builds for every branch that has a PR open.
main is the only branch that needs CI builds (builds regardless of any PR)